dix/req.c

```diff
@@ -1625,13 +1625,32 @@ static void dix_rop_completed(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 	struct m0_dix_rop_ctx *rop_del_phase2 = NULL;
 	bool                   del_phase2 = false;
 	struct m0_dix_cas_rop *cas_rop;
+	uint32_t	       req_cnt = 0;
+	int		       rc;
+	bool		       cas_success = false;
 
 	(void)grp;
 	if (req->dr_type == DIX_NEXT)
 		m0_dix_next_result_prepare(req);
 	else
-		m0_tl_for(cas_rop, &rop->dg_cas_reqs, cas_rop) {
-			dix_cas_rop_rc_update(cas_rop, 0);
+		/*
```

Comment:
Please proofread this comment:

* `status of N CAS requests` or `status of N-th CAS request`?
* Replace `if any one CAS request` with `if any CAS request` or `if there is at least one successful CAS request`.

Reply:
Yes, Agree. 
"Check return status of N CAS requests, if there is at least one successful CAS request" then return success to client. 
should it be fine ?